### PR TITLE
fixed tourist cameras

### LINKED ISSUE #330: Tourist Starts With Three Film Cartridges But No Camera
[Note]: # (State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)

Like the title says, the tourist job starts with plenty of film, but no camera. Not sure if bug or intentional. An admin told me to report it here.

Please fix this, I like the roleplay of the tourist job, taking selfies with all the heads of staff.

### PR DESCRIPTION
**Pull request by ghost**
Fixes #330 
The tourist spawn with a camera film instead a camera in his hands. So, a new tourist arrives to the station with not camera and three camera films. This will fix it. 
:cl:
bugfix: Tourists now spawn with their cameras.
/:cl:

### CODE REVIEW

**Comment by FluffySurvivor** [comment]
wot is the purpose of this pr

**Comment by ghost** [comment]
fixed

**Comment by oisin100** [comment]
Chaaaaaaaaaaaaangeeeeeloooooog

Try adding this to the OP

```
:cl:
bugfix: Tourists now spawn with their cameras.
/:cl:
```

**Comment by Super3222** [comment]
Include "Fixes https://github.com/yogstation13/yogstation/issues/330 (the literal link of the issue)" as the first line please.

File: code/modules/jobs/job_types/civilian.dm [hunk 1]
```diff
@@ -205,7 +205,7 @@ Tourist
 	ears = /obj/item/device/radio/headset
 	belt = /obj/item/device/pda
 	backpack_contents = list(/obj/item/device/camera_film, /obj/item/stack/spacecash/c20, /obj/item/stack/spacecash/c20, /obj/item/stack/spacecash/c20)
-	r_hand =  /obj/item/device/camera_film
+	r_hand =  /obj/item/device/camera // this should fix the problem with the tourist
```
Comment (oisin100) [inline code/modules/jobs/job_types/civilian.dm]: The comment isnt really necessary, Just confusing

Comment (ghost) [inline code/modules/jobs/job_types/civilian.dm]: i just wrote to point where was the problem

**Pull request merged**